rig-qdrant/examples/qdrant_vector_search.rs

```diff
@@ -22,10 +22,9 @@ use rig::{
     Embed,
 };
 use rig_qdrant::QdrantVectorStore;
-use serde_json::json;
 
-#[derive(Embed)]
-struct WordDefinition {
+#[derive(Embed, serde::Deserialize, serde::Serialize, Debug)]
+struct FakeDefinition {
```

Comment:
I had changed the struct to `WordDefinition` since it is implied that these are "fake" and the named was redundant